profile_page/profile_page.css

```diff
@@ -0,0 +1,75 @@
+.profile-main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 70vh;
+  }
+  .profile {
+    max-width: 50rem;
+    width: 100%;
+    border: 2px solid var(--dark-pink);
+    border-radius: 0.4rem;
+    margin: 0 1rem;
+  }
+  .profile-buttons-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-bottom: 2px solid var(--dark-pink);
+    margin: 0 2rem;
+  }
+  .button {
+    margin: 1rem 0.5rem 1rem 0;
+    font-size: 1.1rem;
+    padding: 0.5rem 1.2rem;
+    border-radius: 0.5rem;
+  }
+  .profile-buttons-container a:hover {
+    background-color: var(--dark-pink);
+    color: var(--light-color);
+  }
+  .profile-content-container {
+    display: grid;
+    grid-template-columns: 15.62rem auto;
+  }
+  .user-profile-container {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    margin: 1rem 0.5rem;
```

Comment:
Suggestion: You can use grid-column

Reply:
but I don't understand that shorthand and I tend to make mistake in that case . That is why used this .